app/src/main/java/io/gnosis/safe/ui/safe/terms/TermsBottomSheetDialog.kt

```diff
@@ -0,0 +1,77 @@
+package io.gnosis.safe.ui.safe.terms
+
+import android.text.Html
+import android.text.method.LinkMovementMethod
+import androidx.appcompat.app.AppCompatActivity
+import androidx.lifecycle.Observer
+import com.google.android.material.bottomsheet.BottomSheetDialog
+import io.gnosis.safe.HeimdallApplication
+import io.gnosis.safe.R
+import io.gnosis.safe.di.components.DaggerViewComponent
+import io.gnosis.safe.di.modules.ViewModule
+import io.gnosis.safe.ui.safe.terms.TermsViewModel.ViewAction.ShowBottomSheet
+import io.gnosis.safe.ui.safe.terms.TermsViewModel.ViewAction.TermsAgreed
+import kotlinx.android.synthetic.main.bottom_sheet_terms_and_conditions.*
```

Comment:
can we use view binding and not use synthetics at all?

Reply:
Why not in this case? They are used in several other Fragments and Dialogs.

Reply:
Changed in 47e40600dac8a57c30e85bdd42fcddafd5263afc

Reply:
@elgatovital So as I understand it,: We use view bindings in all the new code. And at some point get rid of the synthetics in existing code?